shared/src/main/scala/kinesis4cats/producer/Producer.scala

```diff
@@ -295,15 +312,18 @@ object Producer {
       * @return
       *   [[kinesis4cats.producer.Producer.Config Producer.Config]]
       */
-    def default(streamNameOrArn: StreamNameOrArn) = Config(
+    def default[F[_]](
+        streamNameOrArn: StreamNameOrArn
+    )(implicit F: Applicative[F]): Config[F] = Config[F](
       true,
       true,
       8,
       false,
       ShardMapCache.Config.default,
       Batcher.Config.default,
       streamNameOrArn,
-      false
+      false,
+      RetryPolicies.alwaysGiveUp[F]
```

Comment:
**suggestion:** maybe we should have retries in the default?

Reply:
I'm not sure I want the library to define a default that likely won't fit most use-cases. This takes the approach of the underlying AWS SDK, in which retries don't occur unless explicitly configured.